README.md

```diff
@@ -377,13 +377,32 @@ console.log(result.js);
 console.log(result.css);
 ```
 
+##### CSS and JS Assets
```

Comment:
I was looking at the entire docs file, and it could be an option to spilt the scope stuff into a separate section, similar to `Events` in order to not have to repeat the same content for `fetch` and `stream` functions.

Reply:
Good idea